### PR TITLE
Update the serverless content for Ruby

### PR DESCRIPTION
**Pull request by fallwith**
Ruby based AWS Lambda functions are now automatically supported by New Relic

### CODE REVIEW

**Comment by fallwith** [comment]
Hi reviewers. I'm on the Ruby agent team.

**Comment by homelessbirds** [comment]
Hi @fallwith would you mind signing the CLA one more time? We've updated it and it requires another signature.

**Comment by fallwith** [comment]
> Hi @fallwith would you mind signing the CLA one more time? We've updated it and it requires another signature.

For employees? Odd. Okay, I've signed it.

**Comment by homelessbirds** [comment]
@fallwith I made a few edits. Can you scan through and make sure I maintained the integrity of the instructions? Thanks!

**Comment by fallwith** [comment]
@homelessbirds I've scanned through and everything looks great. Thanks very much for the fixes!

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-example.mdx [hunk 1]
```diff
@@ -36,6 +36,7 @@ New Relic can instrument:
 * [Python](https://github.com/newrelic/newrelic-lambda-extension/tree/main/examples/sam/python)
 * [Go](https://github.com/newrelic/newrelic-lambda-extension/tree/main/examples/sam/go)
 * [Java](https://github.com/newrelic/newrelic-lambda-extension/tree/main/examples/sam/java)
+* [Ruby](https://github.com/newrelic/newrelic-lambda-extension/tree/main/examples/sam/ruby)
```
Comment (vicentllongo) [inline src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-example.mdx:39]: @fallwith @homelessbirds  This link seems to be broken (There is no example of SAM for Ruby). I landed here from this doc page: https://docs.newrelic.com/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-example/

Comment (fallwith) [reply src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-example.mdx:39]: Hi @vicentllongo. I apologize for the confusion I have caused. There are multiple New Relic teams and PRs involved with the effort to support Ruby based AWS Lambda functions and I ended up submitting this documentation PR and asking for it to be merged too early.

The linked content for Ruby will appear once https://github.com/newrelic/newrelic-lambda-extension/pull/204 has been merged.

In the meantime, our [Ruby Lambda beta test instructions](https://gist.github.com/fallwith/ab8256f447dde65feb2b7e176813cca7) can be used.

**Pull request merged**